fastlane/Fastfile

```
@@ -17,16 +17,17 @@ platform :android do
       task: "assembleBeta",
       properties: { "katana.flavor" => "beta" },
     )
+    commit = last_git_commit
     appcircle_testing_distribution(
       personalAPIToken: ENV["AC_PERSONAL_API_TOKEN"],
       profileName: ENV["AC_PROFILE_NAME"],
       appPath: lane_context[SharedValues::GRADLE_APK_OUTPUT_PATH],
-      message: "Android Beta",
+      message: commit[:message],
```

RuboCop (1.85.1)

[convention] Avoid comma after the last parameter of a method call.

(Style/TrailingCommaInArguments)